lib/model/QubitInformationObject.php

```diff
@@ -74,6 +74,7 @@ public function __get($name)
       case 'formatRegistryName':
       case 'objectUUID':
       case 'aipUUID':
+      case 'aipName':
```

Comment:
This property won't be created in the regular DIP upload (METS based), correct? I'm wondering if it's worth creating it even if it duplicates what's contained in `$aip->filename` just for consistency.

Reply:
Great idea - adding....